component/vshn_postgres.jsonnet

```diff
@@ -392,6 +392,21 @@ local sgCluster = {
   ],
 };
 
+local sgStorageEncryptionSecretObserve = {
+  base: comp.KubeObject('v1', 'Secret') +
+        {
+          spec+: {
+            managementPolicy: 'Observe',
+          },
+        },
+  patches: [
+    comp.ToCompositeFieldPath('status.conditions', 'status.secretConditions'),
+    comp.FromCompositeFieldPathWithTransformPrefix('metadata.labels[crossplane.io/composite]', 'metadata.name', 'secret-observer'),
+    comp.FromCompositeFieldPathWithTransformPrefix('metadata.labels[crossplane.io/composite]', 'spec.forProvider.manifest.metadata.namespace', 'vshn-postgresql'),
+    comp.CombineCompositeFromTwoFieldPaths('metadata.labels[crossplane.io/composite]', 'metadata.labels[crossplane.io/composite]', 'spec.forProvider.manifest.metadata.name', '%s-data-%s-0-luks-key'),
+  ],
+};
+
```

Comment:
Isn't this secret completely managed by the composition function? So we don't really need this right?

Reply:
yes that is true. Removed it again.